sherpa/utils/__init__.py

```diff
@@ -51,8 +51,11 @@
 config = ConfigParser()
 config.read(get_config())
 
-_ncpu_val = config.get('parallel', 'numcores',
-                       fallback='None').strip().upper()
+_ncpu_val = "NONE"
```

Comment:
I don't understand why this has changed - the use of the `fallback` argument with the `get` call means the `try`/`except` block isn't needed. Could this just be because #684 was merged after this code was originally written?